packages/types/src/documents/global/auditLogs.ts

```diff
@@ -15,5 +16,6 @@ export interface AuditLogDoc extends Document {
   timestamp: string
   metadata: any
   name: string
+  type?: "auditLog"
```

Comment:
Should this be optional?

Reply:
Its optional because existing audit logs don't have it - they need to be migrated to have it added. In future it *shouldn't* be optional - but currently all audit logs will be missing this.

I was unsure how to represent this in the types, whether it was better to display what it *should* be, or display what the reality is.

Reply:
I feel like it should always reflect reality, not the ideal. If we go with the ideal we'll end up introducing bugs.